src/pool.c

```diff
@@ -2473,6 +2521,7 @@ client_on_submit(json_object *message, client_t *client)
         share.difficulty = job->target;
         strncpy(share.address, client->address, sizeof(share.address));
         share.timestamp = now;
+        pool_stats.round_hashes += share.difficulty;
```

Comment:
Why call it round hashes when its round accumulated difficulty?

Reply:
diff (it's called job.target in src/pool.c) in miner task description is an estimate of number of hashes to be done by miner to solve task on average.
I use term hashes instead of diff to describe done work by pool workers.

Do you prefer round_diff?

Reply:
I definitely prefer round_diff.

Reply:
I'm going to leave round_hashes.